Stateless/DotGraph.cs

```diff
@@ -0,0 +1,34 @@
+﻿using System.Collections.Generic;
+using System.Linq;
+
+namespace Stateless
+{
+    public partial class StateMachine<TState, TTrigger>
+    {
+        /// <summary>
+        /// A string representation of the state machine in the DOT graph language.
+        /// </summary>
+        /// <returns>A description of all simple source states, triggers and destination states except those defined with PermitDynamic.</returns>

```

Comment:
Is it possible to show some evidence of the existence of dynamic triggers in the resulting graph, even though we can't determine the destination states? I.e. perhaps a pseudo-state called `?` that would be the target of dynamic triggers.. Any other options perhaps?